package.json

```diff
@@ -51,13 +55,19 @@
   },
   "license": "MIT",
   "devDependencies": {
+    "@vuepress/plugin-active-header-links": "^1.2.0",
+    "@vuepress/plugin-medium-zoom": "^1.2.0",
+    "@vuepress/plugin-nprogress": "^1.2.0",
+    "jsdoc-api": "^5.0.4",
+    "vuepress": "^1.2.0",
     "@node-minify/core": "^5.2.1",
     "@node-minify/no-compress": "^5.2.0",
     "@node-minify/terser": "^5.2.1",
     "eslint": "^6.7.2",
-    "jsdoc": "~3.5.5",
     "qunit": "^2.9.3",
     "resemblejs": "^3.2.3",
+    "rollup": "^1.27.9",
```

Comment:
I imagine this is for the new builds. When I `npm install` I get errors on a dependency that I think that version of `rollup` is requesting:

![roll-up](https://user-images.githubusercontent.com/220033/73402262-2eea2280-42ed-11ea-8884-25e56338b2ab.PNG)

Do you get these errors too?

Reply:
I don't. What versions of Node and NPM are you running?